assets/css/_page/_single.scss

```diff
@@ -113,16 +113,16 @@
       }
     }
 
-    > h2 > .header-mark::before {
```

Comment:
Hi @WolfeCub,

Can we use something like this?

```css
   > h2,
   > h3,
   > h4,
   > h5,
   > h6,
   > .outline-text-2 > h2,
   > .outline-text-3 > h3,
   > .outline-text-4 > he,
   > .outline-text-5 > h5,
   > .outline-text-6 > h6,
 {
      > .header-mark::before {
        content: "|";
        margin-right: 0.3125rem;
      }
    }
```

I think it's more performant to use `child selectors` instead of the `descendant selectors`.